templates/common/.devcontainer/devcontainer.json/nodejs/devcontainer.json

```diff
@@ -3,7 +3,7 @@
     "build": {
         "dockerfile": "Dockerfile",
         "args": {
-            "VARIANT": "bullseye"
+            "IMAGE": "javascript-node:16"
```

Comment:
There's also a typescript image, I'm not sure if we'd prefer to use that one.

https://github.com/devcontainers/images/tree/main/src/typescript-node

vs

https://github.com/devcontainers/images/tree/main/src/javascript-node